selfdrive/car/fw_versions.py

```diff
@@ -104,6 +111,18 @@ class Request:
 
 
 REQUESTS: List[Request] = [
+  # Chrysler / FCA / Stellantis
+  Request(
+    "chrysler",
+    [CHRYSLER_VERSION_REQUEST],
+    [CHRYSLER_VERSION_RESPONSE],
```

Comment:
@realfast do you know which Ecus respond to which requests? We have whitelisting support now, so it would be ideal to start off with the correct request whitelists.